{
	"id": "oval:com.redhat.rhsa:def:20193704",
	"version": "636",
	"class": "patch",
	"metadata": {
		"title": "RHSA-2019:3704: numpy security update (Moderate)",
		"reference": [
			{
				"ref_id": "RHSA-2019:3704",
				"ref_url": "https://access.redhat.com/errata/RHSA-2019:3704",
				"source": "RHSA"
			},
			{
				"ref_id": "CVE-2019-6446",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2019-6446",
				"source": "CVE"
			}
		],
		"description": "The numpy packages provide NumPY. NumPY is an extension to the Python programming language, which adds support for large, multi-dimensional arrays and matrices, and a library of mathematical functions that operate on such arrays.\n\nSecurity Fix(es):\n\n* numpy: crafted serialized object passed in numpy.load() in pickle python module allows arbitrary code execution (CVE-2019-6446)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Red Hat Enterprise Linux 8.1 Release Notes linked from the References section.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Moderate",
			"updated": {
				"date": "2019-11-05"
			},
			"cve": [
				{
					"text": "CVE-2019-6446",
					"cvss3": "8.8/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
					"cwe": "CWE-358->CWE-20",
					"href": "https://access.redhat.com/security/cve/CVE-2019-6446",
					"impact": "moderate",
					"public": "20190116"
				}
			],
			"affected": {},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:rhel_aus:8.4",
					"cpe:/a:redhat:rhel_aus:8.4::appstream",
					"cpe:/o:redhat:rhel_aus:8.4",
					"cpe:/o:redhat:rhel_aus:8.4::baseos"
				]
			},
			"rights": "Copyright 2019 Red Hat, Inc.",
			"issued": {
				"date": "2019-11-05"
			},
			"bugzilla": [
				{
					"text": "crafted serialized object passed in numpy.load() in pickle python module allows arbitrary code execution",
					"href": "https://bugzilla.redhat.com/1667950",
					"id": "1667950"
				}
			]
		},
		"affected": {
			"family": "unix",
			"platform": [
				"Red Hat Enterprise Linux 8"
			]
		}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterions": [
							{
								"test_ref": "oval:com.redhat.rhba:tst:20191992003",
								"comment": "Red Hat Enterprise Linux 8 is installed"
							},
							{
								"test_ref": "oval:com.redhat.rhba:tst:20191992004",
								"comment": "Red Hat CoreOS 4 is installed"
							}
						]
					},
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20193704001",
										"comment": "python3-numpy is earlier than 1:1.14.3-9.el8"
									},
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20193704002",
										"comment": "python3-numpy is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20193704003",
										"comment": "python3-numpy-f2py is earlier than 1:1.14.3-9.el8"
									},
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20193704004",
										"comment": "python3-numpy-f2py is signed with Red Hat redhatrelease2 key"
									}
								]
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.rhba:tst:20191992005",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
